cartographer/io/serialization_format_migration.cc

```diff
@@ -295,5 +297,68 @@ void MigrateStreamFormatToVersion1(
                             output);
 }
 
+mapping::MapById<mapping::SubmapId, mapping::proto::Submap>
+MigrateSubmapFormatVersion1ToVersion2(
+    const mapping::MapById<mapping::SubmapId, mapping::proto::Submap>&
+        submap_id_to_submap,
+    mapping::MapById<mapping::NodeId, mapping::proto::Node>& node_id_to_node,
+    const mapping::proto::PoseGraph& pose_graph_proto) {
+  using namespace cartographer::mapping;
```

Comment:
`using mapping;` if you want to be short, or `using ::cartographer::mapping;` for the full namespace?